src/components/ChatLog.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+const ChatLog = ({ entries, updateChat }) => {
+  const makeMessage = entries.map((message) => {
```

Comment:
Style: Since this is a collection it should have a plural name.  Also, `makeMessage` sounds like a function name since it uses an active verb.

```suggestion
  const messages = entries.map((message) => {
```